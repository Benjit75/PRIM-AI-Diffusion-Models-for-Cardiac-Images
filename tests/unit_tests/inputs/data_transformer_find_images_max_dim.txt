{'data_names': ['test'], 'image_types': ['image_data'], 'image_names': ['ES', 'ES_gt']}
{'data_names': ['test', 'train'], 'image_names': ['ED']}
{'data_names': ['train'], 'image_types': ['image_data_resized']}
{'group_names': ['NOR']}