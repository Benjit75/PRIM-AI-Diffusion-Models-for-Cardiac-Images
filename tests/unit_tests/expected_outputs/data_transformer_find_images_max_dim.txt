(15,13)
(16,16)
(0,0)
(16,15)